compiler/aarch64/codegen/OMRCodeGenerator.cpp

```diff
@@ -66,6 +66,7 @@ OMR::ARM64::CodeGenerator::CodeGenerator() :
    self()->setLastGlobalFPR(TR::Machine::getLastGlobalFPRRegisterNumber());
 
    self()->getLinkage()->initARM64RealRegisterLinkage();
+   self()->setSupportsGlRegDeps();
```

Comment:
Don't you call `self()->setSupportsGlRegDepOnFirstBlock()`?  All other platforms call the function, too.

Reply:
I believe  that `setSupportsGlRegDepOnFirstBlock()` is for linkage register allocation as we talked privately.